components/dashboard/script.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+
+const vars = {
+    REACT_APP_HOSTNAME : "Gitpod",
```

Comment:
<details>
<summary>:warning: <strong>This code is using `"` double-quotes to declare strings.</strong> (click for details)</summary>

A possible improvement would be to use `'` single-quote to declare all strings.  
  
This change would make the code consistent and easier to maintain.

**[More details](https://confluence.devfactory.com/x/8BhnFQ)**

## The improvement suggestion in this case
```js  
REACT_APP_HOSTNAME : 'Gitpod',
```  
> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence(s).
</details>

[//]: # (<a id="ab662e3f-0a8e-4170-abb1-881e35e802eb" data-finding-id="359078128" href="none" codereview-uuid-reserved-tag data-business-id="3d94a8d2-7431-4dbe-bce7-80b87bb68d39" data-analyzer="js" />)